packages/web_framework/requirements.txt

```diff
@@ -1,3 +1,4 @@
+asgiref
```

Comment:
Not needed any more?

Reply:
Should be, but let me run the tests again to see why

Reply:
oh, I see. yeah, that was for the coroutine wrangling.